### PR TITLE
Ylva-express-api

### PR DESCRIPTION
**Pull request by sssylvasss**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,32 +1,152 @@
...
+//     }
+//   } else if (artist){
+//     const artistName = topMusicData.filter((item) => {
+//     return item.artistName.toLowerCase().indexOf(artist.toLowerCase()) !== -1;
+// })
+//     res.json({ data: artistName });
+//   } 
+  
+// })
+
```
Comment (KarolCislowski) [inline server.js:49]: what happened here? 🕵🏻

File: server.js [hunk 2]
```diff
@@ -1,32 +1,152 @@
...
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
```
Comment (KarolCislowski) [inline server.js:17]: Nice cleaning 🧹

File: server.js [hunk 3]
```diff
@@ -1,32 +1,152 @@
...
+  let musicToSend = topMusicData;
+  
+  if (title) {
+    musicToSend = musicToSend
+    .filter(music => music.trackName.toLowerCase().includes(title.toLowerCase()));
+  } 
+  if (artist) {
+    musicToSend = musicToSend
+    .filter(music => music.artistName.toLowerCase().includes(artist.toLowerCase()));
+  }
```
Comment (KarolCislowski) [inline server.js:63]: nice filtering 💯

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
Please add a Heroku link for testing 🤓

File: server.js (hunk 1 above)
Comment (sssylvasss) [reply server.js]: What do you mean? It's an endpoint that sort either by song title or artist name. Or do you mean that it is commented out?

**Comment by sssylvasss** [comment]
https://expressylva.herokuapp.com/

**Comment by sssylvasss** [comment]
https://expressylva.herokuapp.com/

**Pull request closed without merging**